ui_framework/src/components/expression/_expression.scss

```diff
@@ -48,12 +48,6 @@
 
     // 1. Angulars ng-hide uses display: none. To use animations we need to use visibility instead.
     &.ng-hide,
```

Comment:
Now `&.ng-hide` shares the block with `&:before`, which is probably not intended. I think the previous block should remain for `&.ng-hide` to avoid a flash of unstyled content.

Reply:
@weltenwort Ok.